backend/translation/app/translate.py

```diff
@@ -1,47 +1,43 @@
-from transformers import pipeline, AutoTokenizer, AutoModelForSeq2SeqLM
-import time
+import json
+import argostranslate.package
+import argostranslate.translate
 
-translate_model1 = "facebook/nllb-200-distilled-600M"
-# translate_model2 = "facebook/nllb-200–1.3B"
-# translate_model3 = "facebook/nllb-200–3.3B"
-# translate_model4 = "facebook/nllb-200-distilled-1.3B"
 
-language_mapped = {
-    "sv": "swe_Latn",
-    "en": "en_Latn",
-    "ru": "rus_Cyrl",
-    "ja": "jpn_Jpan",
-    "fa": "pes_Arab",
-    "zh": "zho_Hans",
-    "es": "spa_Latn",
-    "fr": "fra_Latn",
-}
 
+language_codes = {"en", "sv", "fr", "es", "de", "ar", "zh", "zt", "nl", "fi", "hi", "pl", "ru"}
 
-def translate_to_lang(data: dict, from_language: str, to_language: str) -> list:
-    print(f"Translation started...")
-    start_time = time.time() # TODO only for time print, remove later
-    
-    model = AutoModelForSeq2SeqLM.from_pretrained(translate_model1)
-    tokenizer = AutoTokenizer.from_pretrained(translate_model1)
-    translator = pipeline(
-        "translation",
-        model=model,
-        tokenizer=tokenizer,
-        src_lang=language_mapped.get(from_language),  
-        tgt_lang=language_mapped.get(to_language),    
-        max_length=400                                 
-)
-    
-    translated_data = []
+def install_language(from_language: str, to_language: str):
+    from_code = from_language
```

Comment:
gör inte så lägg in variable namnen som argument direct istället gör att byta den här